sdcm/logcollector.py

```diff
@@ -400,7 +400,9 @@ def dashboard_exists(grafana_ip, uid):
         checked_dashboard_url = "http://{grafana_ip}:{grafana_port}/api/dashboards/db/{uid}"
         try:
             LOGGER.info(
-                "Check dashbaord: %s", checked_dashboard_url.format(grafana_ip=grafana_ip, grafana_port=MonitoringStack.grafana_port, uid=uid))
+                "Check dashbaord: %s", checked_dashboard_url.format(grafana_ip=grafana_ip,
```

Comment:
little typo in the word "dashboard"

```suggestion
                "Check dashboard: %s", checked_dashboard_url.format(grafana_ip=grafana_ip,
```